runtime/compiler/x/runtime/X86Codert.nasm

```diff
@@ -334,6 +334,72 @@ f2l_done:
 
 %endif ; TR_HOST_32BIT
 
+; crc32c
+;
+; Computes CRC32C checksum for a given buffer and size.
```

Comment:
It would be helpful to future readers if you could briefly describe the technique you're about to employ in this procedure in this comment.